examples/passthroughfs.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python3
 '''
+ext filesystem compatibility checked by https://github.com/zfsonlinux/fstest.git
```

Comment:
This is rather a remark or property of this implementation, but nothing the description of this should start with, so move it a bit lower, please.